ircb/cli/user.py

```diff
@@ -17,8 +17,8 @@ def user_cli():
 @click.argument('password', required=False)
 @coroutinize
 def user_create(username, email, password):
-    """Create a user"""
-    yield from UserStore.create(dict(
+	"""Create a user"""
+	yield from UserStore.create(dict(
```

Comment:
Do not use TAB: ``\t``. Always use 4 spaces instead of tabs. You can configure your text editor/IDE accordingly.